scripts/components/Main.js

```diff
@@ -149,6 +152,35 @@ export default class Main extends React.Component {
     });
   }
 
+  initialScoreCard() {
+    const scoreCard = {}
+    for(const sceneId in this.context.params.scenes){
+      const scene = this.context.params.scenes[sceneId];
+      scoreCard[sceneId] = this.initialSceneScoreCard(scene);
+    }
+    return scoreCard;
+  }
+
+  initialSceneScoreCard(scene) {
+    const sceneScoreCard = {
+      title: scene.scenename
+    };
+    for(let i = 0; i < scene.interactions.length; i++){
+      const interaction = scene.interactions[i];
+      switch(interaction.action.library) {
+        case "H5P.Summary 1.10":
+          sceneScoreCard[i]={title: interaction.label.labelText, raw: 0, max: 1, scaled: 0};
+          break;
+        case "H5P.SingleChoiceSet 1.11":
+          sceneScoreCard[i]={title: interaction.label.labelText, raw: 0, max: interaction.action.params.choices.length, scaled: 0};
+          break;
+        default:
+          // Noop
```

Comment:
`H5P.libraryFromString` will parse the library name and give the library's machine name. Using the machine name only, we don't have to update the version numbers later:


```suggestion
      const libraryName = H5P.libraryFromString(interaction.action.library).machineName;
      switch(libraryName) {
        case "H5P.Summary":
          sceneScoreCard[i]={title: interaction.label.labelText, raw: 0, max: 1, scaled: 0};
          break;
        case "H5P.SingleChoiceSet":
          sceneScoreCard[i]={title: interaction.label.labelText, raw: 0, max: interaction.action.params.choices.length, scaled: 0};
          break;
        default:
          // Noop
```

Reply:
Why was this resolved?